hasher-matcher-actioner/hmalib/lambdas/aws_secrets.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import boto3
+import base64
+import typing as t
+import os
+import ast
+from botocore.exceptions import ClientError
+
+from threatexchange.api import ThreatExchangeAPI
+
+AWS_REGION = os.environ["AWS_REGION"]
+
+class AWSSecrets():
+    """
+    A class for reading secrets stored in aws for interacting with ThreatExchange
+    """
+    def te_api_data(self) -> "ThreatExchangeAPI" :
+        secret_name = "threatexchange/api_keys"
+        secret_str = get_str_secret(secret_name)
+        return ThreatExchangeAPI(secret_str)
+
+def get_bin_secret(secret_name: str) -> t.BinaryIO:
+    response = get_secret_value_response(secret_name)
+    assert response and 'SecretBinary' in response
+    decoded_binary_secret = base64.b64decode(get_secret_value_response['SecretBinary'])
+    return decoded_binary_secret
+
+def get_str_secret(secret_name: str) -> str:
+    response = get_secret_value_response(secret_name)
+    assert response and 'SecretString' in response
```

Comment:
(opinion - for this and all the other asserts): In general, this can erase some of the context around the error that python might be giving you for free. For example, simple doing line 32 will lead to a "KeyError: SecretString" pointing to the line, which is actually more descriptive than the error that will be displayed for the assert failure.

This is one reason you don't see too many asserts in python code.